mmdet/datasets/utils.py

```diff
@@ -133,6 +133,11 @@ def _check_head(self, runner):
             for name, module in model.named_modules():
                 if hasattr(module, 'num_classes') and not isinstance(
                         module, (RPNHead, VGG, FusedSemanticHead, GARPNHead)):
+                    assert type(dataset.CLASSES) is not str, \
```

Comment:
This part of code is valuable and necessary. Can we move it to another place for a more clear logic and only check it once? For example, move it to line 133 before delving into each modules.

Reply:
moved outside for loop